clustered/server/build.gradle

```diff
@@ -17,18 +17,18 @@
 apply plugin: EhDeploy
 
 dependencies {
-  compile project(':clustered:common'), "org.slf4j:slf4j-api:$slf4jVersion"
-  compile("org.terracotta:offheap-resource:$terracottaPlatformVersion") {
+  api "org.terracotta:client-message-tracker:$terracottaPlatformVersion"
+  api("org.terracotta:offheap-resource:$terracottaPlatformVersion") {
     transitive = false
   }
-  compile group: 'org.terracotta', name: 'offheap-store', version: offheapVersion
-  compile group: 'org.slf4j', name: 'slf4j-api', version: slf4jVersion
-  compile("org.terracotta.management:monitoring-service-api:$terracottaPlatformVersion") {
+  implementation project(':clustered:common')
+  implementation group: 'org.terracotta', name: 'offheap-store', version: offheapVersion
+  implementation group: 'org.slf4j', name: 'slf4j-api', version: slf4jVersion
+  implementation ("org.terracotta.management:monitoring-service-api:$terracottaPlatformVersion") {
     transitive = false
   }
-  compile"org.terracotta.management.dist:mnm-common:$terracottaPlatformVersion"
-  compile("org.terracotta:client-message-tracker:$terracottaPlatformVersion")
-  provided "org.terracotta:entity-server-api:$terracottaApisVersion"
-  provided "org.terracotta:standard-cluster-services:$terracottaApisVersion"
-  provided "org.terracotta:runnel:$terracottaPlatformVersion"
+  implementation "org.terracotta.management.dist:mnm-common:$terracottaPlatformVersion"
+  providedImplementation "org.terracotta:entity-server-api:$terracottaApisVersion"
```

Comment:
Totally unrelated but why is there a 's' on 'Api' in 'terracottaApisVersion'?

Reply:
Done.

Reply:
I guess the person who added it considered the fact it applied to multiple things to warrant a plural.

Reply:
It was because the GitHub project is named `terracotta-oss/terracotta-apis`